.github/workflows/london_taxi_pr_pipeline.yml

```diff
@@ -31,7 +31,8 @@ jobs:
       exec_environment: ${{ inputs.exec_environment || 'pr' }}
       model_type: ${{ inputs.model_type || 'london_taxi' }}
     secrets:
-      subscription_id: ${{ secrets.SUBSCRIPTION_ID }}
-      resource_group_name: ${{ secrets.RESOURCE_GROUP_NAME }}
-      workspace_name: ${{ secrets.WORKSPACE_NAME }}
-      azure_credentials: ${{ secrets.AZURE_CREDENTIALS }}
+      subscription_id: ${{ vars.SUBSCRIPTION_ID }}
```

Comment:
secrets to envs

Reply:
Can they remain as variables or do they have to be environment variables.